src/main/java/com/google/opensesame/projects/ProjectData.java

```diff
@@ -55,54 +53,16 @@ public static ProjectData fromProjectEntity(ProjectEntity projectEntity, GHRepos
    * @return Returns the mentor mock data.
    * @throws IOException
    */
-  private static List<MentorObject> createMentorMockData() throws IOException {
-    List<MentorObject> mentors = new ArrayList<MentorObject>();
-
-    ArrayList<String> ObiSkills = new ArrayList<String>();
-    ObiSkills.add("Meme god");
-    ObiSkills.add("HTML wrangler");
-    MentorObject Obi =
-        new PersonBuilder()
-            .name("Obi")
-            .gitHubID("Obinnabii")
-            .description("Obi is awesome.")
-            .interestTags(ObiSkills)
-            .buildMentor();
-    mentors.add(Obi);
-
-    ArrayList<String> SamiSkills = new ArrayList<String>();
-    SamiSkills.add("Stone carver");
-    SamiSkills.add("Bootstrap convert");
-    MentorObject Sami =
-        new PersonBuilder()
-            .name("Sami")
-            .gitHubID("Sami-2000")
-            .description("Sami is fun.")
-            .interestTags(SamiSkills)
-            .buildMentor();
-    mentors.add(Sami);
-
-    ArrayList<String> RichiSkills = new ArrayList<String>();
-    RichiSkills.add("Minecraft boss");
-    RichiSkills.add("React wizard");
-    MentorObject Richi =
-        new PersonBuilder()
-            .name("Richi")
-            .gitHubID("Richie78321")
-            .description("Richi is cool.")
-            .interestTags(RichiSkills)
-            .buildMentor();
-    mentors.add(Richi);
-
-    return mentors;
+  private static List<UserData> createMentorMockData() throws IOException {
+    return null;
   }
```

Comment:
Do we have plans to re-implement this? If so, let's please add a TODO.

If not, we can delete this method and simply use null directly in line 46.